autodp/cdf_bank.py

```diff
@@ -0,0 +1,93 @@
+"""
+The work flow of AFA: https://arxiv.org/abs/2106.08567)
```

Comment:
Add some comments about what you expect functions in this cdf_bank to do.   Which random variable should this CDF be for?